### PR TITLE
Updating size recommendations for CIM

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/25544

### CODE REVIEW

File: multicluster_engine/cluster_lifecycle/create_infra_env.adoc [hunk 1]
```diff
@@ -78,11 +78,11 @@ spec:
...
 +
-Replace `img_volume_size` with the size of the volume for the `imageStorage` field, for example `10GB`. This value specifies how much storage is allocated for the images of the clusters. You need to allow 1 GB of image storage for each instance of Red Hat Enterprise Linux CoreOS that is running. You might need to use a higher value if there are many clusters and instances of Red Hat Enterprise Linux CoreOS.
+Replace `img_volume_size` with the size of the volume for the `imageStorage` field, for example `2G` per operating system image. The size minimum is `50G`. This value specifies how much storage is allocated for the images of the clusters. You need to allow 1 GB of image storage for each instance of Red Hat Enterprise Linux CoreOS that is running. You might need to use a higher value if there are many clusters and instances of Red Hat Enterprise Linux CoreOS.
```
Comment (jelkosz) [inline multicluster_engine/cluster_lifecycle/create_infra_env.adoc]: "The size minimum" should probably be "The minimum size"

Comment (dockerymick) [reply multicluster_engine/cluster_lifecycle/create_infra_env.adoc]: great catch, thanks!

File: multicluster_engine/cluster_lifecycle/create_infra_env.adoc [hunk 2]
```diff
@@ -78,11 +78,11 @@ spec:
       cpuArchitecture: "x86_64"
 ----
 +
-Replace `db_volume_size` with the volume size for the `databaseStorage` field, for example `50M`. This value specifies how much storage is is allocated for storing files like database tables and database views for the clusters. You might need to use a higher value if there are many clusters.
+Replace `db_volume_size` with the volume size for the `databaseStorage` field, for example `10G`. This value specifies how much storage is is allocated for storing files like database tables and database views for the clusters. You might need to use a higher value if there are many clusters.
 +
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/create_infra_env.adoc:82]: such as in place of "like"

**Review by swopebe — APPROVED** [review]
one small recommendation on content that you didn't write in this PR.

**Comment by jelkosz** [comment]
lgtm

**Comment by dockerymick** [comment]
Thanks for the review @swopebe and @jelkosz! merging the PR

**Pull request merged**